packages/manager/apps/procedures/src/pages/create/form/Form.page.tsx

```diff
@@ -104,7 +104,7 @@ const FormCreateRequest = () => {
         </div>
       )}
 
-      {legalForm && (
+      {(legalForm || legalForm?.trim()?.length === 0) && (
```

Comment:
I think it would be best to test:
`(!isOtherLegalFormForFR || legalForm)`
This way when we are in a nominal case (which doesn't require to let user select his/her legal form), even if the `legalForm` is empty we would display the `FormDocumentFieldList`.
And when we are in a non nominal case (we have to display the legalForm dropdown) we won't display the form until the user has selected one